### PR TITLE
Update browserstack.md url traffic identifiers

### PR DESCRIPTION
**Pull request by michal-at-travisci**
As requested by BrowserStack - adjusted URLs in Travis CI documentation

### CODE REVIEW

File: user/browserstack.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ layout: en
 
 ---
 
-Travis CI integrates with [BrowserStack](https://www.browserstack.com), a cross browser and real device
+Travis CI integrates with [BrowserStack](https://www.browserstack.com/?utm_source=rapise&utm_medium=partnered), a cross browser and real device
```
Comment (Mitesh-07) [inline user/browserstack.md:7]: Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered

File: user/browserstack.md [hunk 2]
```diff
@@ -15,17 +15,17 @@ and BrowserStack servers. Local Testing also has support for firewalls, proxies
 Once the secure connection is setup, all URLs work out of the box, including your webserver, local folders, as well as
 URLs with HTTPS.
 
-[local-testing]: https://www.browserstack.com/local-testing
+[local-testing]: https://www.browserstack.com/local-testing/?utm_source=rapise&utm_medium=partnered
```
Comment (Mitesh-07) [inline user/browserstack.md:18]: Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered

File: user/browserstack.md [hunk 3]
```diff
@@ -15,17 +15,17 @@ and BrowserStack servers. Local Testing also has support for firewalls, proxies
...
 
-[local-binary]: https://www.browserstack.com/local-testing#command-line
+[local-binary]: https://www.browserstack.com/local-testing#command-line/?utm_source=rapise&utm_medium=partnered
```
Comment (Mitesh-07) [inline user/browserstack.md:20]: Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered

File: user/browserstack.md [hunk 4]
```diff
@@ -15,17 +15,17 @@ and BrowserStack servers. Local Testing also has support for firewalls, proxies
...
 
-[open-source-browserstack]: https://www.browserstack.com/pricing
+[open-source-browserstack]: https://www.browserstack.com/open-source/?utm_source=rapise&utm_medium=partnered
```
Comment (Mitesh-07) [inline user/browserstack.md:22]: Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered

File: user/browserstack.md [hunk 5]
```diff
@@ -15,17 +15,17 @@ and BrowserStack servers. Local Testing also has support for firewalls, proxies
...
 
-[account-settings]: https://www.browserstack.com/accounts/settings
+[account-settings]: https://www.browserstack.com/accounts/settings/?utm_source=rapise&utm_medium=partnered
```
Comment (Mitesh-07) [inline user/browserstack.md:24]: Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered

File: user/browserstack.md [hunk 6]
```diff
@@ -15,17 +15,17 @@ and BrowserStack servers. Local Testing also has support for firewalls, proxies
...
 
 [encryption-keys]: https://docs.travis-ci.com/user/encryption-keys/
 
-[browserstack-ruby-bindings]: https://www.browserstack.com/automate/ruby
+[browserstack-ruby-bindings]: https://www.browserstack.com/automate/ruby/?utm_source=rapise&utm_medium=partnered
```
Comment (Mitesh-07) [inline user/browserstack.md:28]: Hi @michal-at-travisci , could you please change the UTM link to this: https://www.browserstack.com/?utm_source=travis&utm_medium=partnered

**Review by qasim-at-tci — APPROVED** [review]

**Pull request merged**